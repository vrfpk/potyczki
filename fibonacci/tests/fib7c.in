234124960153600
