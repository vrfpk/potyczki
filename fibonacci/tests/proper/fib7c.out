2010612989644800
